lostpixel.base.config.ts

```diff
@@ -0,0 +1,11 @@
+import { CustomProjectConfig } from 'lost-pixel';
```

Comment:
We have a lot of configs in the project root. Probably, we need additional task to move some of them to the common directory ("config" for example)